### ISSUE TITLE
Export as CSV doesn't work?

### ISSUE THREAD

**Issue by FrocketGaming** [tgrosinger/advanced-tables-obsidian#204]
I'm not sure if it's working as intended or not but when you do Export as CSV it just brings up a box with the table data and a checkbox to include headers or not. There is no 'export' functionality to it. It is supposed to have a button to actually save the table as CSV?

**Comment by himcraft** [comment]
U need manually copy the contents in the popup box and paste them into a new CSV file.

**Comment by jimjam-slam** [comment]
I'm on Advanced Tables 0.19.1 (Obsidian 1.5.0), and the export dialog contains a TSV export (except with the table header separators retained) rather than a CSV export:

![image](https://github.com/tgrosinger/advanced-tables-obsidian/assets/6520659/0da003b3-096c-4d14-b9d7-9e41cbae1732)

If you uncheck "Include table headers", both the headers themselves and the header separators disappear (first two rows of the export).

**Comment by qizhengyang2017** [comment]
> I'm on Advanced Tables 0.19.1 (Obsidian 1.5.0), and the export dialog contains a TSV export (except with the table header separators retained) rather than a CSV export:
> 
> ![image](https://private-user-images.githubusercontent.com/6520659/284424155-0da003b3-096c-4d14-b9d7-9e41cbae1732.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzEwNzUzMTgsIm5iZiI6MTczMTA3NTAxOCwicGF0aCI6Ii82NTIwNjU5LzI4NDQyNDE1NS0wZGEwMDNiMy0wOTZjLTRkMTQtYjlkNy05ZTQxY2JhZTE3MzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDhUMTQxMDE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjNhNWU3OWRmZjM4NjY2NWNiZTdjMzY3NTNjNzM1NWJhODU3NGI0YTk3NDIwMjJhZWZkMTc0ZjhmYzYzZmZmYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.VbHmXw9ZYhI3KamhSVn3S037eHTtOADxKV7vZpPX6pc)
> 
> If you uncheck "Include table headers", both the headers themselves and the header separators disappear (first two rows of the export).

The same problem with Advanced Tables 0.22.1 (Obsidian v1.7.5)

**Comment by dbastien** [comment]
Yeah totally not a csv, maybe have options for both tsv (as it is now i guess) and csv.

Although I get this at the top when I get the tsv you get from exporting as csv - is that legit tsv stuff?

Name	Abbreviation	Role	Location	Info
------------------	------------	----------------------------------	-------------------------------------------	-------------------------------------------------------------------------------------------------------------------------------
